src/test/java/core/basesyntax/service/impl/ReportServiceImplTest.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.service.ReportService;
+import org.junit.After;
+import org.junit.Test;
+
+public class ReportServiceImplTest {
+    private ReportService reportService = new ReportServiceImpl();
+
+    @Test
+    public void creatingReport_Ok() {

```

Comment:
Pay attention to names
![image](https://user-images.githubusercontent.com/20326858/215064746-0bd87bc9-64d1-4556-b3f8-9a3d6819f575.png)